### ISSUE TITLE
3x3 sigmoid/softmax output.

### ISSUE THREAD

**Issue by DBAFC** [MrGiovanni/UNetPlusPlus#63]
Hello

I was wondering what the rational is for using 3x3 convolutions as opposed to 1x1 convolutions in the output layer. As far as i know, the original U-net paper uses 1x1 sigmoid/softmax neurons in the output for pixelwise classification, but in you implementation you use 3x3. Why is that?

**Comment by MrGiovanni** [comment]
Hi @DBAFC 

Which implementation did you refer to? I think I used 1x1 convolutions in the output layer as well.

See

**Comment by DBAFC** [comment]
Hi

This is the model i'm referring to:

https://github.com/MrGiovanni/UNetPlusPlus/blob/master/keras/segmentation_models/unet/builder.py
